templates/status.html

```diff
@@ -11,7 +11,7 @@
 </head>
 <body>
     <h1 class="text-center">kube-applier</h1>
-    {{ if .Start }}
+    {{ if .CommitHash }}
```

Comment:
This fixes an unrelated bug. The idea is to show a temporary loading page before the first run results come in. Unfortunately the "Start" field starts with a truthy value. CommitHash starts as empty string, which is falsy.